tamr_toolbox/data_io/dataframe.py

```diff
@@ -317,13 +317,40 @@ def _check_nonnull_columns(
     return ValidationCheck(passed, failed_checks_dict)
 
 
+def _check_custom(
+    df: "pandas.DataFrame",
+    columns_to_check: Optional[List[str]],
+    check_function: Callable[[Any], bool],
```

Comment:
I think this is just
```suggestion
    check_function: Callable[Any, bool],
```

Reply:
I was wrong here and that's why the CI is failing. Sorry! You were correct, the first "arg" to `Callable` should be `[Any]` (a list).